controller/src/main/java/org/jboss/as/controller/persistence/yaml/YamlConfigurationExtension.java

```diff
@@ -207,6 +207,11 @@ private void processResource(PathAddress parentAddress, Map<String, Object> yaml
                     Object value = yaml.get(name);
                     if (value instanceof Map) {
                         Map<String, Object> map = (Map<String, Object>) value;
+                        if (value != null &&
```

Comment:
L208 makes this value != null check unnecessary.

Reply:
@ehsavoie This review has not been resolved yet in the code.

Reply:
sorry I did it when I rebased WFCORE-6348 over it